src/auro-dropdown.js

```diff
@@ -215,6 +215,31 @@ class AuroDropdown extends LitElement {
     this.dispatchEvent(event);
   }
 
+  /**
+   * @param {Object} event - Event passed in from the document click event listener that this function gets attached to.
+   * @returns {void} Method used to close the dropdown bib when clicking in the view outside the dropdown.
+   */
+  offClick(event) {
```

Comment:
Naming is hard. `offClick` kind of confuses me. At first, I was assuming that this meant something like a mouseup event. 

Popular names I remember were along the lines of `outsideClick`.